src/gwt/src/org/rstudio/studio/client/workbench/views/packages/ui/InstallPackageDialog.java

```diff
@@ -73,6 +74,8 @@ public InstallPackageDialog(
       server_ = server;
       globalDisplay_ = globalDisplay;
 
+      RStudioThemes.disableDarkMenus();
```

Comment:
Fair enough, changed. This also had to be reworked in a few other ways since I found more dialogs with elements of dark menus and transitions, should be all good now.